include/tvm/runtime/device_api.h

```diff
@@ -47,10 +47,10 @@ enum DeviceAttrKind : int {
 };
 
 /*! \brief Number of bytes each allocation must align to */
-constexpr int kAllocAlignment = 64;
+constexpr int kAllocAlignment = 128;
```

Comment:
Could there be any issues with increasing the alignment block across all devices @tqchen

Reply:
This should be fine